test.sh

```diff
@@ -9,8 +9,41 @@ trap finish EXIT
 
 function main() {
   docker-compose build --pull tester
+  local secret='
+{
+  "a": 1,
+  "b": "xyz"
+}
+'
+  export AWS_ACCESS_KEY_ID=x
+  export AWS_SECRET_ACCESS_KEY=x
+  export AWS_TEST_ENDPOINT=http://localstack:4584
+  export AWS_DEFAULT_REGION=us-east-1
+
+  docker-compose up -d localstack
+  echo "waiting for localstack to be ready"
+  until $(docker-compose exec -T localstack wget -qO /dev/null "http://localhost:8080/"); do
+    sleep 2;
+    printf ".";
+  done
+  echo "done"
 
-  docker-compose run --rm \
+  docker-compose exec -T \
+   localstack \
+   bash -c "
+printenv AWS_DEFAULT_REGION
```

Comment:
You should indent the bash commands so that they line up as children of the `bash -c` for easier reading